package-lock.json

```diff
@@ -16,11 +16,11 @@
       "integrity": "sha512-lZEV1ZANzfzSYBU6WHSErsy7jLPbD1iIgAboASPMcKo7woVni5/5IKWeT0RxC8rY802MFktur3OKEw2JY1Tv2w==",
       "dev": true,
       "requires": {
-        "@babel/plugin-proposal-async-generator-functions": "7.2.0",
-        "@babel/plugin-proposal-dynamic-import": "7.5.0",
```

Comment:
Is there a reason you prefer the package-lock to not lock to a specific version?

Reply:
i didnt do this on purpose it's an artifact of me running `wrangler preview`.. not sure what the deal is i can revert these changes

Reply:
package-lock stuff is so weird 😤 do you mind removing these changes from this PR?

Reply:
yes! done :)